packages/turf-point-on-line/index.js

```diff
@@ -86,20 +87,25 @@ module.exports = function (line, pt, units) {
         if (intersect) {
             intersectPt = point(intersect);
             intersectPt.properties.dist = distance(pt, intersectPt, units);
+            intersectPt.properties.location = length + distance(start, closestPt, units);
         }
 
         if (start.properties.dist < closestPt.properties.dist) {
             closestPt = start;
             closestPt.properties.index = i;
+            closestPt.properties.location = length + distance(start, closestPt, units);
```

Comment:
I'm pretty sure you can drop the `distance()` function here since `closestPt === start` therefore always being a distance of zero.

```javascript
closestPt.properties.location = length;
```